eth2deposit/utils/ssz.py

```diff
@@ -42,17 +42,15 @@ def compute_signing_root(ssz_object: Serializable, domain: bytes) -> bytes:
     return domain_wrapped_object.hash_tree_root
 
 
-# DepositMessage SSZ
-
-class DepositMessage(Serializable):
+class UnsignedDeposit(Serializable):
     fields = [
         ('pubkey', bytes48),
         ('withdrawal_credentials', bytes32),
         ('amount', uint64),
     ]
 
 
-class Deposit(Serializable):
```

Comment:
These renamings look good in this repo, but the inconsistency to the spec may not be good. IMO it's easier to maintain it by aligning with the spec.

Reply:
Personally, I think the names are very confusing, but I agree that reflecting the spec is a good idea. I changed this to match the spec in commit d96670d below